app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -84,11 +82,9 @@ fun KaigiApp(
     modifier: Modifier = Modifier,
 ) {
     KaigiTheme {
-        val systemUiController = rememberSystemUiController()
         val useDarkIcons = !isSystemInDarkTheme()
 
-        DisposableEffect(systemUiController, useDarkIcons) {
-            systemUiController.setSystemBarsColor(Color.Transparent, useDarkIcons)
+        DisposableEffect(useDarkIcons) {
             onDispose {}
         }
```

Comment:
Maybe we can remove this? 👀

Reply:
That's true. It do nothing :sweat_smile: 
I removed it !
https://github.com/DroidKaigi/conference-app-2023/pull/910/commits/2e408f9b579bbd9b92858e101f628a4c03c90a25